src/introduction/startup-flow.md

```diff
@@ -0,0 +1,33 @@
+# Application Startup Flow
+
+In order to boot an application, Espressif devices use 2 bootloaders:
+- First Stage Bootloader (ROM Bootloader): Setups architecture-specific registers, checks the [boot mode][boot-mode] and reset reason, and loads the second stage bootloader.
+- Second Stage Bootloader: Loads the [partition table][partition-table] and your application.
+
+The second stage bootloader is required as it allows OTA support (first stage bootloader only load applications from a fixed offset in Flash) and also for Flash encrytion and secure boot.
+
+For more information about the startup process, please check [ESP-IDF documentation][esp-idf-startup]. However, note that some aspects may be ESP-IDF specific.
+
+[boot-mode]: https://docs.espressif.com/projects/esptool/en/latest/esp32c6/advanced-topics/boot-mode-selection.html?highlight=boot%20mode
+[partition-table]: #partition-tables
+[esp-idf-startup]: https://docs.espressif.com/projects/esp-idf/en/stable/esp32c6/api-guides/startup.html
+
+### Second Stage Bootloader Alternatives
```

Comment:
Not sure what the intended meaning is here, if there is only one supported bootloader than there is no alternative 😁